ark/segmentation/signal_extraction.py

```diff
@@ -1,15 +1,15 @@
 import numpy as np
 
 
-def positive_pixels_extraction(cell_coords, image_data, threshold=0):
+def positive_pixels_extraction(cell_coords, image_data, **kwargs):
     """Extract channel counts by summing over the number of non-zero pixels in the cell.
 
-    Improves on default_extraction by not distinguishing between pixel expression values
+    Improves on total_intensity_extraction by not distinguishing between pixel expression values
```

Comment:
I think we can delete this little bit of editorializing

Reply:
Thoughts on 'A New York Times best extractor, it's the highly anticipated sequel to **total_intensity_extraction**'?